pages/ourteam.js

```diff
@@ -0,0 +1,180 @@
+import React, { Component } from 'react';
+import NavbarBootstrap from '../components/NavbarBootstrap';
+import Card from 'react-bootstrap/Card';
+import Jumbotron from 'react-bootstrap/Jumbotron';
+import Image from 'react-bootstrap/Image';
+import Figure from 'react-bootstrap/Figure';
+import Container from 'react-bootstrap/Container';
+import Col from 'react-bootstrap/Col';
+import Row from 'react-bootstrap/Row';
+import Button from 'react-bootstrap/Button';
+import Background from '../img/background.jpeg';
+import axios from 'axios';
+import './ourteam.css';
+
+class Ourteam extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+        isAuthenticated: false,
+        name: '',
+    };
+  }
+
+  componentDidMount = () => {
+      const authToken = window.localStorage.getItem('auth_token');
+
+      if (authToken) {
+          axios.post(`${host}:${port}/authstatus`, { headers: { 'Authorization' : `Bearer ${authToken}` } }).then(res => {
+              if (res.status === 200) {
+                  this.setState({ isAuthenticated: true, name: res.data.authData.name });
+              }
+          }).catch(err => {});
+      }
+  }
+
+
+  render() {
+    return (
+
+      <div style={{ backgroundImage: `url(${Background})`, maxWidth: '100%', height: '100%' }}>
+          <link
+              rel="stylesheet"
+              href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+              integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+              crossorigin="anonymous"
+          />
+          <link
+              rel="stylesheet"
+              href="https://use.fontawesome.com/releases/v5.0.13/css/all.css"
+              integrity="sha384-DNOHZ68U8hZfKXOrtjWvjxusGo9WQnrNx2sqG0tfsghAvtVlRW3tvkXWZh58N9jp"
+              crossorigin="anonymous"
+          />
+          <link
+              href="https://fonts.googleapis.com/css?family=Raleway"
+              rel="stylesheet"
+          />
+          <header>
+            <NavbarBootstrap
+                isAuthenticated={false}
```

Comment:
this prop should not be static `false`, it should be based on the value of `this.state.isAuthenticated`

Reply:
you also need to pass in the name prop